docs/getting-started.txt

```diff
@@ -1,300 +1,338 @@
-#################
- Getting Started
-#################
+#########################
+ Quickstart for ML Users
+#########################
 
-This section walks through the steps to run examples in Determined from zero. You should be able to:
+This quickstart walks you through the basic steps to install and run Determined, train on the
+provided model data, and visualize training experiment results in your browser.
 
-#. Get a light touch with Determined concepts and features;
-#. Install Determined;
-#. Get familiar with the WebUI and the CLI;
-#. Train an example model with a single GPU;
-#. Train an example model with distributed training;
-#. Train an example model with hyperparameter tuning.
+Three model data and code examples use the MNIST dataset to demonstrate:
 
-*********************
- System Requirements
-*********************
+-  training on a local, single CPU with or without a GPU.
+-  how to run a distributed training job on multiple GPUs to improve performance.
+-  how to increase model accuracy using hyperparameter tuning.
 
-Please review :doc:`this document </sysadmin-deploy-on-prem/requirements>` to see if your system
-meets the requirements to run Determined, including instructions on how to install the Docker.
+These are followed by a demonstration of how Determined seamless integrates with TensorFlow.
 
-****************************
- Install Determined Locally
-****************************
+Follow these steps for a hands-on introduction to Determined. After gaining familiarity with
+Determined tools and procedures, replace the provided model data and code with your model data and
+code, including integrating with the Determined APIs.
```

Comment:
This is to walk users through the steps of running code on Determined. We should point them to use examples and model hub here.